synology_api/downloadstation.py

```diff
@@ -105,6 +105,14 @@ def tasks_info(self, task_id, additional_param=None):
 
         return self.request_data(api_name, api_path, req_param)
 
+    def create_task(self, uri):
```

Comment:
As there are many method would you add create_uri_task to be more specific on what the function does

Reply:
Sure, but I just named it after the official API doc: https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/DownloadStation/All/enu/Synology_Download_Station_Web_API.pdf#%5B%7B%22num%22%3A56%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C466%2C376%2C0%5D

The param can be either HTTP/FTP/magnet/ED2K links or the file path, so I can maybe name it create_download_task and name the param "link"

Reply:
Ok so its tested that works with files? Because u did not test it yet as in not home